qemu/qemu-guest-agent/qemu-guest-agent.yaml

```diff
@@ -0,0 +1,51 @@
+name: qemu-guest-agent
+depends:
+  - service: cri
+  - path: /dev/virtio-ports/org.qemu.guest_agent.0
+container:
+  entrypoint: ./qemu-ga
+  mounts:
+    # Shared libraries.
+    - source: /lib
+      destination: /lib
+      type: bind
+      options:
+        - bind
+        - ro
+    - source: /usr/lib
+      destination: /usr/lib
+      type: bind
+      options:
+        - bind
+        - ro
+    # State files.
+    - source: /var/run
+      destination: /var/run
+      type: bind
+      options:
+        - rshared
+        - rbind
+        - rw
+    # Device files.
+    - source: /dev
+      destination: /dev
+      type: bind
+      options:
+        - rshared
+        - rbind
+        - rw
+    # `/sbin/init` talks to `apid`.
+    - source: /system/run/apid/apid.sock
+      destination: /system/run/apid/apid.sock
+      type: bind
+      options:
+        - rshared
+        - rbind
+        - rw
```

Comment:
this could be `ro` since socket files has no concept or read/write

Reply:
Interesting, do you not need write access to connect to a socket?

I tried it with `ro` and it does work, but it is quite surprising.

Reply:
> I tried it with ro and it does work, but it is quite surprising.

probably something to do with the fact that init talks to it, but in general file permissions does not makes sense for socket files (if it doesn;'t work with ro keep rw for now)